src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/UnprotectedAttributeTests.cs

```diff
@@ -268,6 +270,37 @@ private static void VerifyUnprotectedAttributes_Heterogenous3(byte[] encodedMess
         }
 
         [Fact]
+        [ActiveIssue(9252, PlatformID.Any)]
```

Comment:
Since you made this test work (by changing the expected value to 1) you should clear the active issue flag.